.github/workflows/dockerbuild.yml

```diff
@@ -0,0 +1,32 @@
+name: CD-Build
+
+on:
+  push:
+    branches:
+      - master
+    paths:
+      - '.github/workflows/dockerbuild.yml'
+      - 'website/**'
+
+jobs:
+  cd:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v1
+      - name: Declare some variables
+        id: vars
+        shell: bash
+        run: |
+          echo "##[set-output name=branch;]$(echo ${GITHUB_REF#refs/heads/})"
+          echo "::set-output name=sha_short::$(git rev-parse --short HEAD)"
+      - name: Docker login
+        run: echo ${{ secrets.DOCKER_PASSWORD }} | docker login -u ${{ secrets.DOCKER_USER }} --password-stdin
+      - name: Build
+        run: docker build -t docusaurusdocs .
+      - name: Tags
+        run: |
+          docker tag docusaurusdocs jonsy13/docusaurusdocs:${{ steps.vars.outputs.branch }}-${{ steps.vars.outputs.sha_short }}
+      - name: Push
+        run: |
+          docker push jonsy13/docusaurusdocs:${{ steps.vars.outputs.branch }}-${{ steps.vars.outputs.sha_short }}
```

Comment:
can we take the docker repo/tag also as env vars

Reply:
https://docs.github.com/en/free-pro-team@latest/actions/reference/environment-variables#default-environment-variables